diff -U3 -r projectM-complete-2.1.0-Source/src/libprojectM/Renderer/MilkdropWaveform.cpp projectM-complete-2.1.0-Source-new//src/libprojectM/Renderer/MilkdropWaveform.cpp
--- projectM-complete-2.1.0-Source/src/libprojectM/Renderer/MilkdropWaveform.cpp	2012-05-20 06:43:26 +0300
+++ projectM-complete-2.1.0-Source-new//src/libprojectM/Renderer/MilkdropWaveform.cpp	2024-11-04 18:07:16 +0300
@@ -10,7 +10,7 @@
 #include <GL/gl.h>
 #endif
 #ifdef WIN32
-#include "glew.h"
+#include <GL/glew.h>
 #endif
 #ifdef __APPLE__
 #include <OpenGL/gl.h>
diff -U3 -r projectM-complete-2.1.0-Source/src/libprojectM/Renderer/TextureManager.cpp projectM-complete-2.1.0-Source-new//src/libprojectM/Renderer/TextureManager.cpp
--- projectM-complete-2.1.0-Source/src/libprojectM/Renderer/TextureManager.cpp	2012-05-20 06:43:26 +0300
+++ projectM-complete-2.1.0-Source-new//src/libprojectM/Renderer/TextureManager.cpp	2024-11-04 18:07:28 +0300
@@ -2,7 +2,7 @@
 #include <GL/gl.h>
 #endif
 #ifdef WIN32
-#include "glew.h"
+#include <GL/glew.h>
 #endif
 #ifdef __APPLE__
 #include <OpenGL/gl.h>
diff -U3 -r projectM-complete-2.1.0-Source/src/libprojectM/Renderer/Waveform.cpp projectM-complete-2.1.0-Source-new//src/libprojectM/Renderer/Waveform.cpp
--- projectM-complete-2.1.0-Source/src/libprojectM/Renderer/Waveform.cpp	2012-05-20 06:43:26 +0300
+++ projectM-complete-2.1.0-Source-new//src/libprojectM/Renderer/Waveform.cpp	2024-11-04 18:07:39 +0300
@@ -9,7 +9,7 @@
 #include <GL/gl.h>
 #endif
 #ifdef WIN32
-#include "glew.h"
+#include <GL/glew.h>
 #endif
 #ifdef __APPLE__
 #include <OpenGL/gl.h>
