diff -U3 -r wavpack-5.0.0-old/src/open_filename.c wavpack-5.0.0-new/src/open_filename.c
--- wavpack-5.0.0-old/src/open_filename.c	2016-12-05 22:46:14 +0300
+++ wavpack-5.0.0-new/src/open_filename.c	2016-12-25 11:33:30 +0300
@@ -66,11 +66,11 @@
 
 static int64_t get_pos (void *id)
 {
-#ifdef _WIN32
-    return _ftelli64 ((FILE*) id);
-#else
+//#ifdef _WIN32
+//    return _ftelli64 ((FILE*) id);
+//#else
     return ftell ((FILE*) id);
-#endif
+//#endif
 }
 
 static int set_pos_abs (void *id, int64_t pos)
