From: =?utf-8?q?Rog=C3=A9rio_Theodoro_de_Brito?= <rbrito@gmail.com>
Date: Sun, 22 Nov 2020 20:05:27 -0300
Subject: Skip directories and style checks for flake8

Forwarded: no
Last-Update: 2020-04-10
---
 Makefile | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

Index: yt-dlp-2021.08.10/Makefile
===================================================================
--- yt-dlp-2021.08.10.orig/Makefile
+++ yt-dlp-2021.08.10/Makefile
@@ -46,7 +46,10 @@ install: yt-dlp yt-dlp.1 completions
 	install -Dm644 completions/fish/yt-dlp.fish $(DESTDIR)$(SHAREDIR)/fish/vendor_completions.d/yt-dlp.fish
 
 codetest:
-	flake8 .
+	flake8 \
+		--exclude=.svn,CVS,.bzr,.hg,.git,__pycache__,.tox,.eggs,*.egg,.pc,.pybuild \
+		--ignore=E121,E123,E126,E226,E24,E704,E402,E501,E731,E741,W503,W504,N802,N803,N806,N816,F401,F821,E302,E305,N801,N815,E265,N813,F403,F405 \
+		.
 
 test:
 	$(PYTHON) -m pytest
