Index: yt-dlp-2021.10.10/Makefile
===================================================================
--- yt-dlp-2021.10.10.orig/Makefile
+++ yt-dlp-2021.10.10/Makefile
@@ -8,7 +8,7 @@ tar: yt-dlp.tar.gz
 # Keep this list in sync with MANIFEST.in
 # intended use: when building a source distribution,
 # make pypi-files && python setup.py sdist
-pypi-files: AUTHORS Changelog.md LICENSE README.md README.txt supportedsites completions yt-dlp.1 devscripts/* test/*
+pypi-files: Changelog.md LICENSE README.md README.txt supportedsites completions yt-dlp.1 devscripts/* test/*
 
 .PHONY: all clean install test tar pypi-files completions ot offlinetest codetest supportedsites
 
Index: yt-dlp-2021.10.10/setup.py
===================================================================
--- yt-dlp-2021.10.10.orig/setup.py
+++ yt-dlp-2021.10.10/setup.py
@@ -108,7 +108,6 @@ setup(
     long_description_content_type='text/markdown',
     url='https://github.com/yt-dlp/yt-dlp',
     packages=packages,
-    install_requires=REQUIREMENTS,
     project_urls={
         'Documentation': 'https://yt-dlp.readthedocs.io',
         'Source': 'https://github.com/yt-dlp/yt-dlp',
