Index: yt-dlp-2022.07.18/setup.py
===================================================================
--- yt-dlp-2022.07.18.orig/setup.py
+++ yt-dlp-2022.07.18/setup.py
@@ -135,7 +135,6 @@ setup(
     long_description_content_type='text/markdown',
     url='https://github.com/yt-dlp/yt-dlp',
     packages=packages(),
-    install_requires=REQUIREMENTS,
     python_requires='>=3.6',
     project_urls={
         'Documentation': 'https://github.com/yt-dlp/yt-dlp#readme',
