Index: qmmp-plugin-pack-1.3.0/qmmp-plugin-pack.pri
===================================================================
--- qmmp-plugin-pack-1.3.0.orig/qmmp-plugin-pack.pri
+++ qmmp-plugin-pack-1.3.0/qmmp-plugin-pack.pri
@@ -16,6 +16,8 @@ QT += widgets
 
 QMMP_PLUGIN_PACK_VERSION = 1.3.0
 #CONFIG += SVN_VERSION
+CONFIG += hide_symbols
+CONFIG += c++11
 
 #Install paths
 
Index: qmmp-plugin-pack-1.3.0/CMakeLists.txt
===================================================================
--- qmmp-plugin-pack-1.3.0.orig/CMakeLists.txt
+++ qmmp-plugin-pack-1.3.0/CMakeLists.txt
@@ -35,6 +35,9 @@ set(CMAKE_AUTOMOC ON)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 find_package(Qt5Widgets 5.4.0 REQUIRED)
 find_package(Qt5LinguistTools 5.4.0 REQUIRED)
+set(CMAKE_CXX_VISIBILITY_PRESET hidden)
+set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
+set(CMAKE_CXX_STANDARD 11)
 
 get_target_property(QT_LRELEASE_EXECUTABLE Qt5::lrelease IMPORTED_LOCATION)
 
