Index: qmmp-1.6.9/src/qmmpui/fileloader_p.h
===================================================================
--- qmmp-1.6.9.orig/src/qmmpui/fileloader_p.h
+++ qmmp-1.6.9/src/qmmpui/fileloader_p.h
@@ -23,6 +23,7 @@
 #include <QObject>
 #include <QQueue>
 #include <QStringList>
+#include <QSet>
 #include <QFileInfo>
 #include <QMutex>
 #include <QThread>
@@ -89,7 +90,7 @@ private:
     void addDirectory(const QString &s, PlayListItem *before = nullptr);
     bool checkRestrictFilters(const QFileInfo &info);
     bool checkExcludeFilters(const QFileInfo &info);
-    void removeIgnoredTracks(QList<PlayListTrack *> *tracks, const QStringList &ignoredPaths);
+    void removeIgnoredTracks(QList<PlayListTrack *> *tracks, QSet<QString> *ignoredPaths);
     struct LoaderTask
     {
         QString path;
Index: qmmp-1.6.9/src/qmmpui/fileloader.cpp
===================================================================
--- qmmp-1.6.9.orig/src/qmmpui/fileloader.cpp
+++ qmmp-1.6.9/src/qmmpui/fileloader.cpp
@@ -1,5 +1,5 @@
 /***************************************************************************
- *   Copyright (C) 2006-2022 by Ilya Kotov                                 *
+ *   Copyright (C) 2006-2024 by Ilya Kotov                                 *
  *   forkotov02@ya.ru                                                      *
  *                                                                         *
  *   This program is free software; you can redistribute it and/or modify  *
@@ -167,10 +167,10 @@ void FileLoader::insertPlayList(const QS
     tracks.clear();
 }
 
-void FileLoader::addDirectory(const QString& s, PlayListItem *before)
+void FileLoader::addDirectory(const QString &s, PlayListItem *before)
 {
     QList<PlayListTrack *> tracks;
-    QStringList ignoredPaths;
+    QSet<QString> ignoredPaths;
     QDir dir(s);
     dir.setFilter(QDir::Files | QDir::Hidden);
     dir.setSorting(QDir::Name);
@@ -182,7 +182,7 @@ void FileLoader::addDirectory(const QStr
         {
             QStringList paths;
             tracks.append(processFile(info.absoluteFilePath (), &paths));
-            ignoredPaths.append(paths);
+            ignoredPaths.unite(QSet<QString>::fromList(paths));
         }
 
         if (m_finished)
@@ -194,18 +194,16 @@ void FileLoader::addDirectory(const QStr
 
         if(tracks.count() > 30) //do not send more than 30 tracks at once
         {
-            removeIgnoredTracks(&tracks, ignoredPaths);
+            removeIgnoredTracks(&tracks, &ignoredPaths);
             emit newTracksToInsert(before, tracks);
             tracks.clear();
-            ignoredPaths.clear();
         }
     }
 
     if(!tracks.isEmpty())
     {
-        removeIgnoredTracks(&tracks, ignoredPaths);
+        removeIgnoredTracks(&tracks, &ignoredPaths);
         emit newTracksToInsert(before, tracks);
-        ignoredPaths.clear();
     }
 
     dir.setFilter(QDir::Dirs | QDir::NoDotAndDotDot);
@@ -358,18 +356,21 @@ bool FileLoader::checkExcludeFilters(con
     return !QDir::match(m_settings->excludeFilters() , info.fileName());
 }
 
-void FileLoader::removeIgnoredTracks(QList<PlayListTrack *> *tracks, const QStringList &ignoredPaths)
+void FileLoader::removeIgnoredTracks(QList<PlayListTrack *> *tracks, QSet<QString> *ignoredPaths)
 {
-    if(ignoredPaths.isEmpty())
+    if(ignoredPaths->isEmpty())
         return;
 
     QList<PlayListTrack *>::iterator it = tracks->begin();
     while(it != tracks->end())
     {
-        if(ignoredPaths.contains((*it)->path()))
+        QString path = (*it)->path();
+
+        if(ignoredPaths->contains(path))
         {
             delete (*it);
             it = tracks->erase(it);
+            ignoredPaths->remove(path); //exclude ignored path from checking
         }
         else
         {
