Index: qmmp-1.5.2/src/qmmp/inputsource.cpp
===================================================================
--- qmmp-1.5.2.orig/src/qmmp/inputsource.cpp
+++ qmmp-1.5.2/src/qmmp/inputsource.cpp
@@ -161,7 +161,7 @@ QList<InputSourceFactory *> InputSource:
     {
         if(m_disabledNames.contains(item->shortName()))
             continue;
-        if(item->decoderFactory())
+        if(item->inputSourceFactory())
             list.append(item->inputSourceFactory());
     }
     return list;
