diff -U3 -r adplug-2.3.3-old/Makefile.in adplug-2.3.3-new/Makefile.in
--- adplug-2.3.3-old/Makefile.in	2020-06-10 03:53:20 +0300
+++ adplug-2.3.3-new/Makefile.in	2020-08-15 21:02:54 +0300
@@ -197,7 +197,7 @@
 src_libadplug_la_LINK = $(LIBTOOL) $(AM_V_lt) --tag=CXX \
 	$(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=link $(CXXLD) \
 	$(AM_CXXFLAGS) $(CXXFLAGS) $(src_libadplug_la_LDFLAGS) \
-	$(LDFLAGS) -o $@
+	$(LDFLAGS) -no-undefined -o $@
 am_adplugdb_adplugdb_OBJECTS = adplugdb/adplugdb-adplugdb.$(OBJEXT)
 adplugdb_adplugdb_OBJECTS = $(am_adplugdb_adplugdb_OBJECTS)
 am_test_crctest_OBJECTS = test/crctest-crctest.$(OBJEXT)
