diff -U3 -r libbinio-1.5-old/configure.ac libbinio-1.5-new/configure.ac
--- libbinio-1.5-old/configure.ac	2019-08-07 09:01:14 +0300
+++ libbinio-1.5-new/configure.ac	2020-08-15 19:55:40 +0300
@@ -6,6 +6,7 @@
 #AM_DISABLE_SHARED
 AM_PROG_LIBTOOL
 AC_LANG(C++)
+LT_INIT([win32-dll])
 
 # Check for a sane C/C++ build environment.
 AC_PROG_MAKE_SET
diff -U3 -r libbinio-1.5-old/src/Makefile.am libbinio-1.5-new/src/Makefile.am
--- libbinio-1.5-old/src/Makefile.am	2015-06-21 13:31:24 +0300
+++ libbinio-1.5-new/src/Makefile.am	2020-08-15 19:56:00 +0300
@@ -2,7 +2,7 @@
 
 libbinio_la_SOURCES = binio.cpp binfile.cpp binwrap.cpp binstr.cpp
 
-libbinio_la_LDFLAGS = -version-info 1:0:0
+libbinio_la_LDFLAGS = -version-info 1:0:0 -no-undefined
 
 pkginclude_HEADERS = binio.h binfile.h binwrap.h binstr.h
 
